### DISCUSSION TITLE
Browser Wallets not detected on Firefox

### DISCUSSION THREAD

**Discussion by topocount** [rainbow-me/rainbowkit#1613] [Feedback]
I've had a lot of issues getting my frame extension detected on firefox, when injected as both frame and metamask emulation mode. It is never detected on the new app.ens.domains and I've had issues with other sites as well.

**Comment by 0xmkh** [comment]
@topocount To catch up on this you're saying that you have both metamask and frame installed and when you're trying to connect to the site nothing seems to happen ?

**Reply by topocount** [reply]
No, Frame has a mode where it appears as metamask and uses its interfaces if generic injected wallets aren't supported but metamask is.

I don't actually run metamask.

**Reply by 0xmkh** [reply]
Gotcha! When you're using that it's kind of bugging right ? I'll have a look and see what we can do on our end
